src/content/docs/get-started/index.mdx

```diff
@@ -237,168 +219,67 @@ The Code Sync app redeploys your storefront site whenever you push or merge chan
-    1. **Send** the invitation to give Edge Delivery Services (Helix Admin) read/write access to your folder. Your content folder is now connected to the Edge Delivery Services system.
-
-    </Callouts>
-
-  </TabItem>
-
-</Tabs>
-
-</Task>
-
-<Task>
-
-### Add sample content
-
-Now let's add the boilerplate sample files to your content folder. Choose the tab below that matches your chosen storage service, Google Drive or SharePoint.
+### Add content
 
-<Tabs>
-  <TabItem label="Google Drive">
-    <Aside type="tip" title="Google Drive auto-conversions">
-      The "secret" to converting Word and Excel sample files to Google Docs and Sheets is to enable Uploads in Google Drive's settings. Here's how.
-    </Aside>
+Now let's create and initialize the content side of your storefront. We'll create a new folder and share it with Edge Delivery Services.
+Follow these steps to create your https://da.live content folder.
 
-    <Diagram caption="Add sample Google Drive content.">
-      ![Add sample Google Drive content.](images/add-gd-content.webp)
-    </Diagram>
-
-    <Callouts>
-    1. Open your Google Drive folder.
-    1. Select the ⚙️ **Settings icon** (top-right of the page) and choose **Settings** from the menu.
-    1. Enable the **Uploads** checkbox to convert uploads to Google Docs editor format. Return to your storefront folder.
-    1. Download the boilerplate's [starter-content-commerce.zip](https://github.com/hlxsites/aem-boilerplate-commerce/releases/download/starter-content/starter-content-commerce.zip) file.
-    1. Unzip the file and drag the contents into your folder. This will convert the Word and Excel files to Google Docs and Sheets.
-    </Callouts>
-
-    <Aside type="note" title="Ensure Google Drive Uploads Convert Properly">
-      To ensure Word and Excel files are automatically converted to Google Docs and Sheets, **always upload them inside a folder** rather than as individual files. The Google Drive setting for automatic conversion applies to folder uploads, but not to direct file uploads.
-    </Aside>
-
-  </TabItem>
-
-  <TabItem label="SharePoint">
+<Aside type="note" title="Demo Content">
+If you do not want demo content, you can create a file in https://da.live/#/{org}/{site}/ replacing `{org}` and `{site}` with your own. This will "reserve" the org and site for you.
+</Aside>
 
-    <Callouts>
-      1. Download the boilerplate's [starter-content-commerce.zip](https://github.com/hlxsites/aem-boilerplate-commerce/releases/download/starter-content/starter-content-commerce.zip) file.
-      1. Unzip the file and drag the contents into your SharePoint folder.
-    </Callouts>
+<Callouts>
+1. Open the [demo content clone tool](https://da.live/app/hlxsites/aem-boilerplate-commerce/tools/site-creator/site-creator).
+1. Copy your GitHub repository url to the input field and click **Create site**.
+1. Follow the prompts, and by the end the content will have been copied to your content folder.
```

Comment:
so when I used the site clone tool, it ran the preview and publish for me which I could track by following the messages.